bin/travis-build.bash

```diff
@@ -1,11 +1,10 @@
-#!/bin/bash
 set -e
 
 echo "This is travis-build.bash..."
 
 echo "Installing the packages that CKAN requires..."
 sudo apt-get update -qq
-sudo apt-get install postgresql-$PGVERSION solr-jetty libcommons-fileupload-java:amd64=1.2.2-1
```

Comment:
Any reason to remove libcommons?

Reply:
I removed this package because it was causing travis to fail. Travis was throwing a version error even after i modified the versions. In addition, i did some research about this package and discovered that it improves file upload capabilities in servlets and web applications. I didn't see how that would be useful to this extension. For those two reasons, i removed the package. Are there any instances where this might be useful to this extension, in your opinion?

Reply:
I'm figuring at a later date we want to test notifications of dataset being uploaded. But if its not on our requirements right now, we can ignore.